main.js

```diff
@@ -112,16 +112,19 @@
         //TODO: should have some generic error handling for data
         .catch((err) => console.log(err))
         .then((data) => {
+            
             const records = data.result.records;
             records.forEach((record, i) => {
                 //Collect time of incident from the record
+                var popup = JSONtoTable(record);
                 record.incidentYear = parseInt(record.INCIDENTTIME.substring(0, 4));
                 record.incidentMonth = parseInt(record.INCIDENTTIME.substring(5, 8));
                 record.incidentDay = parseInt(record.INCIDENTTIME.substring(8, 10));
 
                 record.pin = L.marker([record.Y, record.X], { icon: CITY_POLICE_ICON });
                 record.pin.addTo(map)
-                    .bindPopup(`${record.OFFENSES}`);
+                //.bindPopup(`${record.OFFENSES}`);
```

Comment:
Part of the beauty of version control is that it isn't really necessary to leave in commented-out old code as reference. We can just delete this, since the change is documented if we ever need to go back and check the old version.